tests/sources/xml/test_datacite.py

```diff
@@ -359,6 +380,136 @@ def test_datacite_with_attribute_and_subfield_variations_transforms_correctly():
     )
 
 
+def test_get_alternate_titles_success():
+    source_record = create_datacite_source_record_stub(
+        """
+        <titles>
+         <title>The Impact of Maternal Literacy and Participation Programs</title>
+         <title titleType="AlternativeTitle">An Alternative Title</title>
+         <title titleType="Subtitle">Baseline Data</title>
+        </titles>
+        """
+    )
+    assert Datacite.get_alternate_titles(source_record) == [
+        AlternateTitle(value="An Alternative Title", kind="AlternativeTitle"),
+        AlternateTitle(value="Baseline Data", kind="Subtitle"),
+    ]
+
+
+def test_get_alternate_titles_transforms_correctly_if_fields_blank(
+    datacite_record_optional_fields_blank,
+):
+    assert Datacite.get_alternate_titles(datacite_record_optional_fields_blank) == []
+
+
+def test_get_alternate_titles_transforms_correctly_if_fields_missing(
+    datacite_record_optional_fields_missing,
+):
+    assert Datacite.get_alternate_titles(datacite_record_optional_fields_missing) == []
+
+
+def test_get_content_type_success():
+    source_record = create_datacite_source_record_stub(
+        """
+        <resourceType resourceTypeGeneral="Dataset">Survey Data</resourceType>
+        """
+    )
+    assert Datacite.get_content_type(source_record, "abc123") == ["Dataset"]
+
+
+def test_get_content_type_transforms_correctly_if_fields_blank(
+    datacite_record_optional_fields_blank,
+):
+    assert (
+        Datacite.get_content_type(datacite_record_optional_fields_blank, "abc123") == []
+    )
+
+
+def test_get_content_type_transforms_correctly_if_fields_missing(
+    datacite_record_optional_fields_missing,
+):
+    assert (
+        Datacite.get_content_type(datacite_record_optional_fields_missing, "abc123") == []
+    )
+
+
+def test_get_contributors_success():
```

Comment:
I appreciate that the test is fully encapsulated so even if it's big, it's all there rather than having to jump to the fixtures. So no complaints from me!